data-builder/build_synonyms.py

```diff
@@ -20,30 +20,32 @@ def build_synonyms():
     add(data, 'Devotion', ['Positive Spell Power', 'Positive Spellpower'])
     add(data, 'Evocation Focus', ['Evocation Spell DCs'])
     add(data, 'False Life', ['Hit Points', 'Lifeforce', 'Maximum HP', 'Maximum Hit Points', 'maximum hitpoints', 'Vitality', 'your maximum hit points'])
+    add(data, 'Force Spell Crit Damage', ['Force and Physical Spell Crit Damage'])
     add(data, 'Healing Lore', ['Positive Spell Crit Chance', 'Positive Spellcrit Chance'])
+    add(data, 'Negative Spell Crit Damage', ['Negative and Poision Spell Crit Damage'])
```

Comment:
I think this needs to be an affix group instead, since Poison has its own dedicated buffs now. (Also, you have a typo in poison although maybe that's in the game!)